src/solver/optim-model-filler/ComponentFiller.cpp

```diff
@@ -369,40 +755,16 @@ void ComponentFiller::addObjective(
     }
 
     const auto& solverVariables = optimEntityContainer_.getVariables();
-    Optimization::ReadLinearExpressionVisitor visitor(
-                                                      ctx,
-                                                      component_,
-                                                      optimEntityContainer_);
+    V::Visitor visitor(optimEntityContainer_, component_, 168); // TODO TimeSteps
```

Comment:
Here we use the newly-introduced visitor instead of `Optimization::ReadLinearExpressionVisitor`